src/components/Header/Header.module.css

```diff
@@ -1,23 +1,30 @@
 .header {
-  height: 82px;
+  --header-height-sm: 3rem;
+  --header-height-lg: 5rem;
+}
+
+.header {
+  height: var(--header-height-sm);
   background-color: var(--black);
   box-shadow: 1px 1px 5px 0 var(--gray);
   position: fixed;
   display: flex;
   justify-content: space-between;
+  align-items: center;
   top: 0;
   width: 100%;
 }
 
 .home {
   display: inline-block;
   color: var(--white);
-  font-size: 16px;
-  margin-left: 10px;
+  font-size: 0.625rem;
+  line-height: 0.625rem;
+  margin-left: 1rem;
 }
 
 .nav {
-  top: 82px;
+  top: var(--header-height-sm);
```

Comment:
FYI `.nav` is the menu overlay that shows on hamburger click. We want to position its top === to the bottom of the header itself. Aka, the nav menu's top is === the height of the header.